app-frontend/app/src/components/head/logos.ts

```diff
@@ -69,5 +69,18 @@ export const LOGOS: Logo[] = [
     alt: () => "Kotlin Language Logo",
     show: () => true,
     exclusive: false,
-  }
+  },
+  {
+    src: require("./10K.svg"),
+    alt: () => {
+      return `Celebrating 10 Years of Kotlin!`
+    },
+    show: () => {
+      const today = new Date();
+
+      return today.getMonth() === 6 && today.getDate() === 19;
```

Comment:
What about date range? I want to more people enjoy this doodle :)

Reply:
Ok, made a range: minus week before the date and two days after.